### PR TITLE
En 5648 golangci fixes

### PR DESCRIPTION
**Pull request by bogdan-rosianu**

### CODE REVIEW

File: process/block/preprocess/transactions_test.go [hunk 1]
```diff
@@ -194,6 +195,12 @@ func initDataPool() *mock.PoolsHolderStub {
 	return sdp
 }
 
+func TestFzxc(t *testing.T) {
```
Comment (miiu96) [inline process/block/preprocess/transactions_test.go]: what this test do ?

File: process/mock/vmContainerMock.go [hunk 2]
```diff
@@ -12,59 +12,56 @@ type VMContainerMock struct {
...
 		return nil
 	}
-	return V.AddMultipleCalled(keys, vms)
+	return v.AddMultipleCalled(keys, vms)
 }
 
-func (V *VMContainerMock) Replace(key []byte, val vmcommon.VMExecutionHandler) error {
-	if V.ReplaceCalled == nil {
+func (v *VMContainerMock) Replace(key []byte, val vmcommon.VMExecutionHandler) error {
+	if v.ReplaceCalled == nil {
```
Comment (miiu96) [inline process/mock/vmContainerMock.go]: maybe you can refactor here... if v.ReplaceCalled !=nil

File: process/smartContract/process_test.go [hunk 3]
```diff
@@ -2342,17 +2341,17 @@ func TestScProcessor_ProcessSmartContractResultWithData(t *testing.T) {
...
-	result = result + test
-	result = result + sep
-	result = result + test
-	result = result + sep
-	result = result + test
-	result = result + sep
-	result = result + test
-	result = result + sep
-	result = result + test
+	result += test
```
Comment (miiu96) [inline process/smartContract/process_test.go]: maybe you can refactor this sequence with a for loop

File: process/sync/baseForkDetector.go [hunk 4]
```diff
@@ -265,7 +265,7 @@ func (bfd *baseForkDetector) append(hdrInfo *headerInfo) bool {
 	defer bfd.mutHeaders.Unlock()
 
 	hdrInfos := bfd.headers[hdrInfo.nonce]
-	isHdrInfosNilOrEmpty := hdrInfos == nil || len(hdrInfos) == 0
+	isHdrInfosNilOrEmpty := len(hdrInfos) == 0 // no need for nil check, len() for nil returns 0
```
Comment (miiu96) [inline process/sync/baseForkDetector.go]: remove variable and just check if len(hdrInfos) == 0...

File: storage/lrucache/lrucache.go [hunk 5]
```diff
@@ -65,7 +65,7 @@ func (c *LRUCache) RegisterHandler(handler func(key []byte)) {
 // Get looks up a key's value from the cache.
 func (c *LRUCache) Get(key []byte) (value interface{}, ok bool) {
 	v, ok := c.cache.Get(string(key))
-	if ok == false {
+	if !ok {
```
Comment (miiu96) [inline storage/lrucache/lrucache.go]: maybe you can just _return c.cache.Get(string(key))_

File: process/block/preprocess/transactions_test.go (hunk 1 above)
Comment (bogdan-rosianu) [reply process/block/preprocess/transactions_test.go]: removed :D

File: process/mock/vmContainerMock.go (hunk 2 above)
Comment (bogdan-rosianu) [reply process/mock/vmContainerMock.go]: no need for this

File: process/smartContract/process_test.go (hunk 3 above)
Comment (bogdan-rosianu) [reply process/smartContract/process_test.go]: done

File: process/sync/baseForkDetector.go (hunk 4 above)
Comment (bogdan-rosianu) [reply process/sync/baseForkDetector.go]: i'd leave it like this because it's easier to understand

File: storage/lrucache/lrucache.go (hunk 5 above)
Comment (bogdan-rosianu) [reply storage/lrucache/lrucache.go]: done

**Pull request merged**